src/pytorch_lightning/CHANGELOG.md

```diff
@@ -154,11 +154,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed an issue to keep downscaling the batch size in case there hasn't been even a single successful optimal batch size with `mode="power"` ([#14372](https://github.com/Lightning-AI/lightning/pull/14372))
```

Comment:
the remaining ones were not added in the patch release?

Reply:
Good question, @krshrimali?

Reply:
When was it merged? I think it should have gone for v1.7.4, and it got missed for v1.7.5 as we didn't take a note of it:

```bash
(pl-17) ➜  lightning git:(master) gh pr list -s merged -S 'merged:2022-08-30T22:30:00.000Z..2022-08-31T22:00:00.000Z' --json mergedAt,milestone,url,mergeCommit,title --jq 'sort_by(.mergedAt) | reverse | .[] | select((.milestone.title == "pl:1.7.x") or (.milestone.title == null) or (.milestone.title == "all:next")) | [.url, .mergeCommit.oid, .title] | join(" ")' --limit 100

https://github.com/Lightning-AI/lightning/pull/14372 e90ac769d6451b3f363f38d4934277613eb3d539 Reset dataloaders on failure in tuner
# .. and other PRs
```

I'll be on-call next week as well, I'll make sure to include it in the next release. From the next time though, I suggest that we should run the commands again before making the releases just to cross-check that nothing is missed. :)

Reply:
I think they got merged on the same day but after the specified time. I'd suggest checking the last commits in the previous release and the `Fixed` section in the changelog on the master to ensure all of them are included since this section is usually empty after release.

Reply:
Great! In the command above, I always extend the date range beyond the last and target release.

Reply:
> Great! In the command above, I always extend the date range beyond the last and target release.

Yeah, that's the perfect solution. Thank you @awaelchli ❤️

Reply:
let's not forget it for the next release, :)
@krshrimali next release is also yours, so lets create a draft PR and already pick these older commits :)